README.md

```diff
@@ -102,18 +102,26 @@ npm install
 
 ## Faster development building with ic-design-system
 
-If you want to build the components in this repository and use them in [`ic-design-system`](https://github.com/mi6/ic-design-system), you can use `npm link`. This will speed up development because you do not have to generate a compressed npm package, but rather directly use the package after building it locally.
+If you want to build the components in this repository and use them in [`ic-design-system`](https://github.com/mi6/ic-design-system), you can use `npm link`.
+This will speed up development because you do not have to generate a compressed npm package, but rather directly use the package after building it locally.
+You will need to rebuild the package after every change though.
 
 First, build the monorepo packages in the top-level directory of `ic-ui-kit`:
 
 ```console
 npm run build
+# If you are working on a canary package then run:
+npm run build:canary
+# Or
+npm run build:all
 ```
 
 Then, to use the package, make sure you are in the top-level directory of `ic-design-system`. You will need to link the generated build from `ic-ui-kit`, e.g.: 
 
 ```console
 npm link ../path/to/ic-ui-kit/packages/react
+# for canary:
```

Comment:
I don't think referencing canary is needed?

Reply:
Also, the code formatting here will break, check the diff in the README.

This needs to change before merge, as it breaks the code style

Reply:
if you don't then someone might run both when they don't need to.

how do you mean the code style is broken?

<img width="529" alt="image" src="https://github.com/user-attachments/assets/250e4445-372d-431d-b435-acdb3d9d8218">

Reply:
I agree with @evenstensberg. Rather than having comments in the code block i think it would be more consistent styling to break the commands out into individual cloaks with text in between eg:

```console
npm link ../path/to/ic-ui-kit/packages/react
```
For the canary package:

```console
npm link ../path/to/ic-ui-kit/packages/canary-react
```

The same applies for the `run build` commands